.github/workflows/runtests.yml

```diff
@@ -114,53 +116,41 @@ jobs:
       - run: bin/doctest --force-colors
       - run: examples/all.py -q
 
-  # -------------------- Test split 1/2 latest Python -------------- #
-
-  tests1-latest:
-    needs: code-quality
-
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.10'
-      - run: python -m pip install --upgrade pip
-      - run: pip install mpmath
-      - run: bin/test --force-colors --split=1/2
-
-  # -------------------- Test split 2/2 latest Python -------------- #
+  # ------------------------- Test latest Python ------------------- #
 
-  tests2-latest:
+  tests-latest:
     needs: code-quality
 
     runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        group: [1, 2, 3, 4]
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
           python-version: '3.10'
       - run: python -m pip install --upgrade pip
-      - run: pip install mpmath
-      - run: bin/test --force-colors --split=2/2
+      - run: pip install mpmath pytest pytest-split pytest-xdist
+      - run: bin/test --split ${{ matrix.group }}/4
 
   # -------------------- Test Pyodide on node ---------------------- #
 
   test-pyodide:
-    needs: code-quality
-
     runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
       matrix:
-        split: ['1/2', '2/2']
+        group: [1, 2, 3, 4]
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16.x
-      - run: wget -qO- https://github.com/pyodide/pyodide/releases/download/0.20.1a1/pyodide-build-0.20.1a1.tar.bz2 | tar xjf -
+          node-version: 18.x
+      - run: wget -qO- https://github.com/pyodide/pyodide/releases/download/0.21.3/pyodide-build-0.21.3.tar.bz2 | tar xjf -
       - run: python3 setup.py bdist_wheel
-      - run: node bin/test_pyodide.mjs --split=${{ matrix.split }} 2>/dev/null  # ignore node exception
+      - run: node bin/test_pyodide.mjs --group=${{ matrix.group }} 2>/dev/null  # ignore node exception
```

Comment:
This redirect prevents seeing error output. It should be removed permanently.

Reply:
Addressed in https://github.com/sympy/sympy/pull/24291/commits/320026505bf5b94af6a7cdee40dd46fe0e301b3b

Reply:
It appear that this is actually necessary to ensure that the workflow doesn't hang even after pytest has finished running successfully. I have added back in for now in https://github.com/sympy/sympy/pull/24291/commits/578e3b73d177fbeac057bde4a8a84e8cffc3964d.